### ISSUE TITLE
Check if devon4net cobigen guide is valid 

### ISSUE THREAD

**Issue by isandesh1986** [devonfw/cobigen#1442]
Check if devon4net cobigen guide is valid 
https://devonfw.com/website/pages/docs/devon4net.wiki_howto.asciidoc_2.html#howto.asciidoc_devon4net-cobigen-guide

**Comment by quantumfate** [comment]
The firs link in the mentioned guide on [this](https://devonfw.com/website/pages/docs/devonfw-guide_devon4net.wiki_cobiGen.asciidoc.html#devonfw-guide_devon4net.wiki_cobigen.asciidoc_getting-things-ready) is broken.

**Comment by quantumfate** [comment]
TL;DR the guide is not valid anymore because it's outdated 

It took me a while to follow the guide because resources are missing. I adapted the guide inside the devon IDE even though the source I linked refers to installing the dotnet sdk outside the devon IDE on the user's system.

The goal of this guide is to generate files with cobigen. So installing the devon ide along with cobigen is necessary. Installing dotnet on your system doesn't make sense.

I found the [dotnet setup guide](https://github.com/devonfw/devon4net/wiki/environment#setting-up-the-environment) that the mentioned guide is referring to. It instructs the user to setup vscode and .net sdk outside of the devon ide and that shouldn't be done when you intend to use cobigen, right? Because the dotnet sdk will be configured and installed automatically by running the command devon dotnet.

This also makes the step obsolete to run ```dotnet new -i `Devon4Net`.WebAPI.Template``` because this will done by setting up the dotnet sdk in the devon IDE. [Take a look at the dotnet command](https://github.com/devonfw/ide/blob/master/scripts/src/main/resources/scripts/command/dotnet).

After I managed to install the templates with `dotnet new Devon4NetAPI` I noticed cobigen created a new directory upon generation because the existing template directory name that was created by the mentioned command is different than the directory name cobigen writes the output to. I used a workaround by editing the input's directory name matching the cobigen output's directory name. Editing the `x-rootpackage` like mentioned [in the guide](https://devonfw.com/website/pages/docs/devonfw-guide_devon4net.wiki_cobiGen.asciidoc.html#devonfw-guide_devon4net.wiki_cobigen.asciidoc_openapi-file) and [in the cobigen api plugin](https://github.com/devonfw/cobigen/wiki/cobigen-openapiplugin#full-example) does actually not influence this behavior.

It doesn't matter if the output is generated correctly or not because in both cases the input created by `devon dotnet new Devon4NetAPI` has a lot of issues itself but mostly wrong imports.

The errors is the only criteria that needs to be fixed for this guide to be valid. @maybeec @isandesh1986
